sim-lib/Cargo.toml

```diff
@@ -15,7 +15,9 @@ expanduser = "1.2.2"
 serde = { version="1.0.183", features=["derive"] }
 serde_json = "1.0.104"
 bitcoin = { version = "0.30.1", features=["serde"] }
-lightning = { version = "0.0.116" }
+# LDK is on a different version to bitcoin to us, which leads to some issues, import so we can use its types.
```

Comment:
Are there reasons why `bitcoin_ldk` types are being used as opposed to sticking with types from `bitcoin`?

Reply:
Long and painful story that's now resolved, but for prosperity: 

We were relying of a different version of the `bitcoin` dependency to LDK:
* `sim-ln` -> `bitcoin-30`
* `sim-ln` -> `lightning-117` -> `bitocin-29`

We couldn't downgrade `bitcoin-30`, because we're using a feature it it. But to use `lightning-117` (specifically in pathfinding) we needed to provide the public key struct type in `bitcoin-29`. So this alias allowed me to import both types. This was messy and gross and thankfully solved by another LDK release that updated the bitcoin version.